example_directives_test.go

```diff
@@ -19,11 +19,15 @@ type HasRoleDirective struct {
 	Role string
 }
 
+func (h *HasRoleDirective) AllowLocation(l string) bool {
+	return l == "FIELD_DEFINITION"
```

Comment:
As per above, this needs to be a constant, however, we need to probably export only the field definitions that are supported by directives at the moment.